test/Helpers/Utils.js

```diff
@@ -0,0 +1,25 @@
+const packet = require('dns-packet');
```

Comment:
Didn't you just make a new package for this?

Reply:
That will be taken care of in another pull request.